page.hbs

```diff
@@ -35,6 +38,60 @@ into the {body} of the default.hbs template --}}
     </div>
 </main>
 
+
+{{!-- Links to Previous/Next posts --}}
+<aside class="read-next outer">
```

Comment:
This supposed to be here too? I don't see it in the UI?

Reply:
Ah I see its the actual preview cards.

FWIW it's kinda hard to tell which is next/previous